jekyll/_cci2/server/Operator/extending-internal-volumes.adoc

```diff
@@ -93,18 +93,18 @@ gp2 (default)   kubernetes.io/aws-ebs   Delete          WaitForFirstConsumer   t
 Now you may proceed to expanding your volumes.
 
 === Step 2 - Delete the database's stateful set
-In this step, you will delete the stateful set, which controls your database pod. The command below deletes the referenced database's stateful set without deleting the pod. You do not want to delete the pod itself, as this would cause downtime. In the following steps, you will redeploy your stateful set. You might chose to delete one or both stateful sets, depending on which database volumes you wish to expand. The `--cascade=false` flag is most important here.
+In this step, you will delete the stateful set, which controls your database pod. The command below deletes the referenced database's stateful set without deleting the pod. You do not want to delete the pod itself, as this would cause downtime. In the following steps, you will redeploy your stateful set. You might chose to delete one or both stateful sets, depending on which database volumes you wish to expand. The `--cascade=orphan` flag is most important here.
 
 For Postgres:
 [source,bash]
 ----
-kubectl delete sts postgresql --cascade=false
+kubectl delete sts postgresql --cascade=orphan
 ----
 
 For MongoDB:
 [source,bash]
 ----
-kubectl delete sts mongodb --cascade=false
+kubectl delete sts mongodb --cascade=orphan
 ----
 
 === Step 3 - Update the size of the database's PVC
```

Comment:
Is Step 3 still required?  Do we really need to patch the PVC before doing a helm upgrade with the new volume size?

Reply:
Yes, Step 3 is still required.